chat-commands.js

```diff
@@ -2921,13 +2921,17 @@ exports.commands = {
 		if (!user.hasConsoleAccess(connection)) {
 			return this.errorReply("/bash - Access denied.");
 		}
-
+		if (!target) {
+			return this.parse('/help bash');
+		}
 		connection.sendTo(room, "$ " + target);
 		let exec = require('child_process').exec;
 		exec(target, (error, stdout, stderr) => {
 			connection.sendTo(room, ("" + stdout + stderr));
 		});
 	},
+	bashhelp:["/bash [argument] - Carries out execution by running a child process",
+	 	  "Requires console access."],
```

Comment:
I guess that would be for people setting up another server? Also like I stated above this crashes the server for some reason when it can be easily handed. I just implemented help as Zarel said it would be a better way of going about things as compared to err out (which I agree with)